webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -159,11 +159,43 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     }
   }, [settings.filterset, isLoadingSettings]);
 
-  const [selectedExperimentIds, setSelectedExperimentIds] = useState<number[]>([]);
```

Comment:
90% of the time we need an `Array<number>`. We only need something else for the initial load and asynchronously saving the settings so this is a bad tradeoff to make. You're slowing down the common case for an uncommon case.

Reply:
You should set the initial state based on settings otherwise you'll get a blip of incorrect state when navigating to this page from another when when settings are already loaded.

Reply:
✅ will init with settings